prime-router/src/main/kotlin/transport/RedoxTransport.kt

```diff
@@ -12,6 +12,7 @@ import gov.cdc.prime.router.ReportId
 import gov.cdc.prime.router.TransportType
```

Comment:
So, fixing RedoxTransport was complicated, first because there was no try/catch block at all, but mainly because each individual send in Redox can fail, and we need to carefully keep track of all of this, both for Retry purposes and also for lineage history.